src/legacy/ui/public/vislib/lib/dispatch.js

```diff
@@ -91,6 +91,33 @@ export function VislibLibDispatchProvider(Private, config) {
       };
     }
 
+    /**
+     * Determine whether rendering a series is configured in percentage mode
+     * Used to display a value percentage formatted in it's popover
+     *
+     * @param rawId {string} The rawId of series to check
+     * @param series {Array} Array of all series data
+     * @param config {VisConfig}
+     * @returns {Boolean}
+      */
+    isSeriesInPercentageMode(rawId, series, config) {
```

Comment:
nit: Looks like the convention here is to prefix "private" functions with an underscore.

Reply:
nit: seems we use rawId only to find the correct series. what if we rather pass in the correct series already ?
```
isSeriesInPercentageMode(series.find(series => series.rawId = d.seriesId), config) ?
```

Reply:
@flash1293 : yes
@ppisljar: doing it this way could cause an Exception, since series could be undefined, so we'd need additional logic on this level